The tests don't run unless DYLD_LIBRARY_PATH is pointed to the directory
where the bundled libCppUnit.dylib and other locally built libraries are.
--- Crypto/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:18.000000000 -0600
+++ Crypto/testsuite/CMakeLists.txt	2021-04-14 21:33:38.000000000 -0500
@@ -27,7 +27,7 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND Crypto-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Crypto PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Crypto PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(Crypto-testrunner PUBLIC Poco::NetSSL Poco::XML CppUnit)
 if(UNIX AND NOT ANDROID)
--- Data/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:19.000000000 -0600
+++ Data/testsuite/CMakeLists.txt	2021-04-14 21:33:30.000000000 -0500
@@ -32,6 +32,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND Data-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Data PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Data PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(Data-testrunner PUBLIC Poco::Data CppUnit)
--- Data/MySQL/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:19.000000000 -0600
+++ Data/MySQL/testsuite/CMakeLists.txt	2021-04-14 22:35:51.000000000 -0500
@@ -23,6 +23,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND DataMySQL-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(DataMySQL PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(DataMySQL PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(DataMySQL-testrunner PUBLIC Poco::DataMySQL CppUnit)
--- Data/ODBC/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:19.000000000 -0600
+++ Data/ODBC/testsuite/CMakeLists.txt	2021-04-14 22:36:00.000000000 -0500
@@ -23,6 +23,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND DataODBC-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(DataODBC PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(DataODBC PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(DataODBC-testrunner PUBLIC Poco::DataODBC CppUnit)
--- Data/PostgreSQL/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:19.000000000 -0600
+++ Data/PostgreSQL/testsuite/CMakeLists.txt	2021-04-14 22:36:11.000000000 -0500
@@ -16,5 +16,5 @@
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 	COMMAND DataPostgreSQL-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 )
-set_tests_properties(DataPostgreSQL PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+set_tests_properties(DataPostgreSQL PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 target_link_libraries(DataPostgreSQL-testrunner PUBLIC Poco::DataPostgreSQL CppUnit)
--- Data/SQLite/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:19.000000000 -0600
+++ Data/SQLite/testsuite/CMakeLists.txt	2021-04-14 22:36:04.000000000 -0500
@@ -27,6 +27,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND DataSQLite-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(DataSQLite PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(DataSQLite PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(DataSQLite-testrunner PUBLIC Poco::DataSQLite CppUnit)
--- Encodings/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:15.000000000 -0600
+++ Encodings/testsuite/CMakeLists.txt	2021-04-14 21:33:23.000000000 -0500
@@ -27,7 +27,7 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND Encodings-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Encodings PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Encodings PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 
 target_link_libraries(Encodings-testrunner PUBLIC Poco::Encodings CppUnit)
--- Foundation/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:13.000000000 -0600
+++ Foundation/testsuite/CMakeLists.txt	2021-04-14 21:22:31.000000000 -0500
@@ -37,7 +37,7 @@
 		WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
 		COMMAND Foundation-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Foundation PROPERTIES ENVIRONMENT "LD_LIBRARY_PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")	# The SharedLibaryTest has to look for shared libraries in the working directory
+	set_tests_properties(Foundation PROPERTIES ENVIRONMENT "LD_LIBRARY_PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")	# The SharedLibaryTest has to look for shared libraries in the working directory
 	set_property(TEST Foundation APPEND PROPERTY ENVIRONMENT "PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}:$ENV{PATH}") # The ProcessTest has to look for the TestApp in the working directory
 	set_property(TEST Foundation APPEND PROPERTY ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR}")
 	# The test is run in the runtime directory. So the test data is copied there too
--- JSON/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:16.000000000 -0600
+++ JSON/testsuite/CMakeLists.txt	2021-04-14 21:33:07.000000000 -0500
@@ -27,7 +27,7 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND JSON-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(JSON PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(JSON PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 	# The test is run in the build directory. So the test data is copied there too
 	add_custom_command(
 		TARGET JSON-testrunner POST_BUILD
--- JWT/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:18.000000000 -0600
+++ JWT/testsuite/CMakeLists.txt	2021-04-14 21:32:58.000000000 -0500
@@ -27,7 +27,7 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND JWT-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(JWT PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(JWT PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(JWT-testrunner PUBLIC Poco::JWT Poco::Crypto CppUnit)
 if(UNIX AND NOT ANDROID)
--- MongoDB/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:20.000000000 -0600
+++ MongoDB/testsuite/CMakeLists.txt	2021-04-14 21:32:45.000000000 -0500
@@ -26,6 +26,6 @@
 		NAME MongoDB
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR} COMMAND MongoDB-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(MongoDB PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(MongoDB PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(MongoDB-testrunner PUBLIC Poco::MongoDB CppUnit)
--- Net/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:17.000000000 -0600
+++ Net/testsuite/CMakeLists.txt	2021-04-14 21:32:35.000000000 -0500
@@ -27,6 +27,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND Net-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Net PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Net PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(Net-testrunner PUBLIC Poco::Net Poco::Util Poco::XML CppUnit)
--- NetSSL_OpenSSL/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:18.000000000 -0600
+++ NetSSL_OpenSSL/testsuite/CMakeLists.txt	2021-04-14 21:32:25.000000000 -0500
@@ -27,7 +27,7 @@
 		WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
 		COMMAND NetSSL-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(NetSSL PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(NetSSL PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 	# The test is run in the build directory. So the test data is copied there too
 	add_custom_command(
 		TARGET NetSSL-testrunner POST_BUILD
--- Redis/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:20.000000000 -0600
+++ Redis/testsuite/CMakeLists.txt	2021-04-14 21:32:14.000000000 -0500
@@ -27,7 +27,7 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND Redis-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Redis PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Redis PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 
 target_link_libraries(Redis-testrunner PUBLIC ${CMAKE_THREAD_LIBS_INIT}	 Poco::Redis CppUnit)
--- Util/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:16.000000000 -0600
+++ Util/testsuite/CMakeLists.txt	2021-04-14 21:31:55.000000000 -0500
@@ -34,6 +34,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND Util-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Util PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Util PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(Util-testrunner PUBLIC Poco::Util Poco::JSON Poco::XML CppUnit)
--- XML/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:15.000000000 -0600
+++ XML/testsuite/CMakeLists.txt	2021-04-14 21:31:44.000000000 -0500
@@ -27,6 +27,6 @@
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 		COMMAND XML-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(XML PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(XML PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endif()
 target_link_libraries(XML-testrunner PUBLIC Poco::XML CppUnit)
--- Zip/testsuite/CMakeLists.txt.orig	2020-02-17 05:01:20.000000000 -0600
+++ Zip/testsuite/CMakeLists.txt	2021-04-14 21:31:30.000000000 -0500
@@ -26,7 +26,7 @@
 		NAME Zip
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR} COMMAND Zip-testrunner -ignore ${CMAKE_SOURCE_DIR}/cppignore.lnx -all
 	)
-	set_tests_properties(Zip PROPERTIES ENVIRONMENT POCO_BASE=${CMAKE_SOURCE_DIR})
+	set_tests_properties(Zip PROPERTIES ENVIRONMENT "POCO_BASE=${CMAKE_SOURCE_DIR};DYLD_LIBRARY_PATH=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 	# The test is run in the build directory. So the test data is copied there too
 	add_custom_command(
 		TARGET Zip-testrunner POST_BUILD
